angular-ngc/defs.bzl

```diff
@@ -294,4 +315,31 @@ def _unit_tests(name, tests, deps, visibility):
         visibility = ["//visibility:private"],
     )
 
-    # TODO: create '{name}' target to actually run/test the tests
+    esbuild(
+        name = "_test_entry_bundle",
+        testonly = 1,
+        entry_points = [spec.replace(".ts", ".js") for spec in test_entry],
```

Comment:
`test_entry` is not a spec. There's also only one so we can use `entry_point`. I also don't think we have to do any compiling or bundling of zonejs though.

Reply:
As we discussed, for now, we have to compile and bundle the `zone.js` in this way, we may make improvement later.